src/Schema/Values/Tab.ts

```diff
@@ -5,10 +5,9 @@
  */
 
 export type Tab =
-  | "cityGuide"
-  | "favorites"
-  | "home"
-  | "messages"
-  | "profile"
-  | "search"
-  | "sell"
+  /**
```

Comment:
Quick note here, tapping 'inbox' icon on the footer tab navigation currently fires with tab type = "messages" but I find that naming inferior to "inbox" -- so data team will have to change historic track data, but I think it's worthwhile.